src/main/java/com/microfocus/application/automation/tools/model/FilterTestsModel.java

```diff
@@ -0,0 +1,117 @@
+package com.microfocus.application.automation.tools.model;
```

Comment:
If this is a new class:
1. Move it under UFT folder
2. Add licenese
3. Add documentation above the class explaining its purpose

Reply:
Also, is this checked on pipeline as well?